hal/inc/gpio_hal.h

```diff
@@ -73,6 +73,8 @@ PinMode hal_gpio_get_mode(hal_pin_t pin);
 void hal_gpio_write(hal_pin_t pin, uint8_t value);
 int32_t hal_gpio_read(hal_pin_t pin);
 uint32_t hal_gpio_pulse_in(hal_pin_t pin, uint16_t value);
+hal_gpio_drive_t hal_gpio_get_drive_strength(hal_pin_t pin);
```

Comment:
Nit: If we are later to expose this in dynalib might perhaps be better to do `int hal_gpio_get_drive_strength(hal_pin_t hal_gpio_drive_t* drive)` to catch errors (e.g. wrong pin, input only pin etc).

Reply:
Fixed.